Cabal/src/Distribution/Verbosity.hs

```diff
@@ -330,15 +333,15 @@ isVerboseMarkOutput = isVerboseFlag VMarkOutput
 isVerboseNoWrap :: Verbosity -> Bool
 isVerboseNoWrap = isVerboseFlag VNoWrap
 
--- | Test if we had called 'lessVerbose' on the verbosity
+-- | Test if we had called 'lessVerbose' on the verbosity.
 isVerboseQuiet :: Verbosity -> Bool
 isVerboseQuiet = vQuiet
 
 -- | Test if we should output timestamps when we log.
 isVerboseTimestamp :: Verbosity -> Bool
 isVerboseTimestamp = isVerboseFlag VTimestamp
 
--- | Test if we should output to stderr when we log.
+-- | Test if we should output to 'stderr' when we log.
```

Comment:
I think this won't be linked because `stderr` isn't in scope (same in the other places in this file). `@stderr@` maybe? The rest looks good.

Reply:
Indeed, did not get linked. But result is the same as with `@stderr@`.